index.html

```diff
@@ -29,7 +29,8 @@ <h2 id="project_tagline">A pairing and mentoring group for software engineers (R
       <section id="main_content" class="inner">
         <h2>
 <a id="start-here" class="anchor" href="#start-here" aria-hidden="true"><span class="octicon octicon-link"></span></a>Start Here</h2>
-
+<p>Please bring your own laptop to work on. Just getting started with Ruby, Rails, Python or Node.JS? We'll help you set up your environment so you can get started!</p>
+<p>Breakfast snacks like begals and fruit are provided.</p>
```

Comment:
If you can please fix this typo, I'll go ahead and merge when you're done. :shipit: